README.md

```diff
@@ -257,6 +258,62 @@ The configfile is currently the only input. The configfile at `.github/auto-merg
 | `[].match.update_type`       | string | true     | Which changes should be merged. Currently supports `semver:patch`, `semver:minor` and `semver:major`. The type includes all lower tiers, for example `semver:minor` includes patch changes |
 | `[].match.package_ecosystem` | string | true     | The name of the package ecosystem which the update belongs to, examples; terraform, golang                                                                                                 |
 
+## update-skip-docs
+
+Automatically creates a pull request for updating documentation on skip.kartverket.no when documentation changes are pushed.
+
+### Features
+
+- Syncs changes to documentation files in the source repository to target in skip.kartverket.no
+- Automatically creates or updates documentation in the target repository
+- Creates a branch with the changes in the target repository
+- Requires manual merge of the created branch/PR to allow for review
+
+### Requirements
+
+- The workflow needs write permissions for both `contents` and `id-token`
+- An STS trust file needs to be configured for the target repository
+- The specified identity must have access to the target repository
+
+### Example
+
+Example usage in `.github/workflows/update-docs.yml`:
+```yaml
+name: Update documentation
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'README.md'
+
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  update-docs:
+    name: Update documentation in target repository
+    uses: kartverket/github-workflows/.github/workflows/update-skip-docs.yaml@latest
+    with:
+      source_path: "README.md"
+      target_path: "docs/03-applikasjon-utrulling/10-argokit/09-argokit-v2.md"
+      branch_name: "update-argokitv2-docs-branch"
+      author_name: "GithubActions <argokit@kartverket.no>"
+      identity: "argokit"
+```
+
+### Inputs
+
+| Key           | Type   | Required | Description                                                                                      |
+|---------------|--------|----------|--------------------------------------------------------------------------------------------------|
+| source_path   | string | X        | Path to the documentation file in the source repository                                          |
```

Comment:
This repository is inconsistent in documentation formatting, but I think using code formatting for key and type improves readability. E.g., `source_path` and `key`.

Different workflows use "X" or "true" for the required field, I find "true"/"false" more explicit and readable.